terraform/digitalocean/main.tf

```diff
@@ -60,26 +60,35 @@ module "etcd_cert" {
   ca_private_key_pem = "${module.ca.ca_private_key_pem}"
 }
 
-module "apiserver_certs" {
-  source             = "../certs/kubernetes/apiserver"
-  ca_cert_pem        = "${module.ca.ca_cert_pem}"
-  ca_private_key_pem = "${module.ca.ca_private_key_pem}"
-  ip_addresses       = "${join(",", digitalocean_droplet.master.*.ipv4_address)}"
-  master_count       = "${var.masters}"
-}
-
-module "worker_certs" {
-  source             = "../certs/kubernetes/worker"
-  ca_cert_pem        = "${module.ca.ca_cert_pem}"
-  ca_private_key_pem = "${module.ca.ca_private_key_pem}"
-  ip_addresses       = "${join(",", digitalocean_droplet.worker.*.ipv4_address)}"
-  worker_count       = "${var.workers}"
-}
-
-module "admin_cert" {
-  source             = "../certs/kubernetes/admin"
-  ca_cert_pem        = "${module.ca.ca_cert_pem}"
-  ca_private_key_pem = "${module.ca.ca_private_key_pem}"
+module "kube_apiserver_certs" {
+  source                = "github.com/Capgemini/tf_tls/kubernetes/apiserver"
+  ca_cert_pem           = "${module.ca.ca_cert_pem}"
+  ca_private_key_pem    = "${module.ca.ca_private_key_pem}"
+  ip_addresses          = "${join(",", digitalocean_droplet.master.*.ipv4_address)}"
```

Comment:
That fails with ```* module 'kube_apiserver_certs': multi-variable must be in a slice``` 

I can navigate round the above issue by passing 
```
ip_addresses  = "${compact(digitalocean_droplet.worker.*.ipv4_address)}"
``` 
not sure that's the best DX, but it works